groovy/material/PetrochemistryMaterials.groovy

```diff
@@ -368,7 +394,11 @@ public class PetrochemistryMaterials {
                 .flags(FLAMMABLE)
                 .build();
 
-        //FREE ID: 20069
+        ClarifiedSlurryOil = new Material.Builder(20069, SuSyUtility.susyId('clarified_slurry_oil'))
```

Comment:
Why did clarified slurry oil get moved, of all things?

Reply:
cause im pretty sure no one will have made it yet

Reply:
also cause i realized i had 1 less id than i needed